Essentials/src/main/java/com/earth2me/essentials/utils/LocationUtil.java

```diff
@@ -267,7 +267,7 @@ public static boolean shouldFly(IEssentials ess, final Location loc) {
         int y = (int) Math.round(loc.getY());
         final int z = loc.getBlockZ();
         int count = 0;
-        while (LocationUtil.isBlockUnsafe(ess, world, x, y, z) && y > -1) {
+        while (LocationUtil.isBlockUnsafe(ess, world, x, y, z) && y <= ess.getWorldInfoProvider().getMinHeight(world)) {
```

Comment:
equal to or greater than moment??

Reply:
copium

Reply:
I just downvoted your comment.

# FAQ

## What does this mean?

The amount of karma (points) on your comment and GitHub account has decreased by one.

## Why did you do this?

There are several reasons I may deem a comment to be unworthy of positive or neutral karma. These include, but are not limited to:

* Rudeness towards other GitHubers,
* Spreading incorrect information,
* Sarcasm not correctly flagged with a `/s`.

## Am I banned from the GitHub?

No - not yet. But you should refrain from making comments like this in the future. Otherwise I will be forced to issue an additional downvote, which may put your commenting and posting privileges in jeopardy.

## I don't believe my comment deserved a downvote. Can you un-downvote it?

Sure, mistakes happen. But only in exceedingly rare circumstances will I undo a downvote. If you would like to issue an appeal, shoot me a private message explaining what I got wrong. I tend to respond to GitHub PMs within several minutes. Do note, however, that over 99.9% of downvote appeals are rejected, and yours is likely no exception.

## How can I prevent this from happening in the future?

Accept the downvote and move on. But learn from this mistake: your behavior will not be tolerated on Reddit.com. I will continue to issue downvotes until you improve your conduct. Remember: Reddit is privilege, not a right.

Reply:
This PR breaks my workflow! My setup works for me. Just add an option to reenable spacebar heating.